react-app/src/components/GovernanceInfoPanel/GovernanceInfoPanel.tsx

```diff
@@ -5,6 +5,29 @@ import Paper from "../common/Paper/Paper";
 import { GovParams, useGovAPI } from "../../api/govAPI";
 import { useLocale } from "../../providers/AppLocaleProvider";
 import LoadingSpinner from "../common/LoadingSpinner/LoadingSpinner";
+import LocalizedText from "../common/Localized/LocalizedText";
+import { Icon, IconType } from "../common/Icons/Icons";
+import { convertBigNumberToFixedPointString } from "../../utils/number";
+import { MessageID } from "../../i18n/LocaleModel";
+
+const InfoField: React.FC<{
+  labelMessageID: MessageID;
+  value: React.ReactNode;
+}> = ({ labelMessageID, value }) => (
+  <div>
+    <h3
+      className={cn(
+        "text-likecoin-lightgreen",
+        "text-sm",
+        "font-medium",
+        "mb-2"
+      )}
+    >
+      <LocalizedText messageID={labelMessageID} />
+    </h3>
+    <div className={cn("text-base", "leading-5", "font-medium")}>{value}</div>
+  </div>
+);
 
 const GovernanceInfoPanel: React.FC = () => {
   const govAPI = useGovAPI();
```

Comment:
Not sure if we should can api within the ui component

Reply:
This component is used in multiple screens, the API is depend purely on RPC and has no dependency with grahpql. I feel like this way would make the component more reusable. 

Let me know if you think I should move the it up to the screen level component?